Software/Development/SemesterProject/App/analogAcquisitionManager.c

```diff
@@ -263,12 +351,16 @@ void createAnalogSignals(void)
     int filterOrder = 1, filterFreq = 50;
 
     // Definition of thresholds
-    float currentThreshold[2] = {0.3, 3.0};
-    float voltageThreshold[2] = {0.0, 3.1};
-    float thermalThreshold[2] = {0.0, 2.5};
-    float sliderPotThreshold[2] = {0.2, 3.0};
-    float rotaryPotThreshold[2] = {0.0, 3.0};
-    float connectorPotThreshold[2] = {0.0, 3.0};
+    float currentThreshold[2], thermalThreshold[2];
+    float DCLinkVoltageThreshold[2], controlVoltageThreshold[2];
+    float sliderPotThreshold[2] = {0.0, 3.3};
+    float rotaryPotThreshold[2] = {0.0, 3.3};
+    float connectorPotThreshold[2] = {0.0, 3.3};
+
+    setCurrentThresholds(&currentThreshold[0], 300, -300);

```

Comment:
Add a comment on (pointer to threshold , MAX VALUE, MIN VALUE)

Reply:
Comment added